sky/services/rasterizer/rasterizer.mojom

```diff
@@ -0,0 +1,12 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
```

Comment:
I don't think we need this annotation because this service isn't accessible in Dart.